docs/user_manual/site_configuration.md

```diff
@@ -0,0 +1,191 @@
+scientific name is entered. You can browse the taxonomy database using NCBI's
+[Taxonomy Browser](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi).
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Automatic Barcode Generation" %}
+
+If automatic barcode generation is enabled for your site, MISO will automatically generate a unique barcode any time
+you leave the barcode field blank for an item. These barcodes, whether auto-generated or not, may be printed from MISO
+to a label printer. See the [Barcode Label Printers section](barcode_label_printers.html) for more information.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Barcode Scanners" %}
+
+Barcode scanners may be used to enter barcodes into any text-entry area in MISO. Most barcode scanners allow you to
+configure a key to send after scanning. If you configure your barcode scanners to send the Enter key after a scan, that
+will save you from having to press Enter or click on a button or the next item every time to scan a barcode. This
+configuration is done on the barcode scanner itself, and you should see the scanner's manual for configuration
+instructions.
+
+It is also possible to use
+[Thermo Fisher VisionMate™ High Speed Barcode Readers](https://www.thermofisher.com/order/catalog/product/312800) to
+scan entire boxes of tubes at once. These bulk scanners must be configured by your MISO administrators to work in MISO.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Report Links" %}
+
+{% assign figure = figure | plus: 1 %}
+If you are generating reports for projects or runs outside of MISO, it may be possible to automatically link to them
+from the MISO Project and Run pages. For projects, it must be possible to generate the report's URL given the project's
+ID, name, and short name. For runs, it must be possible to generate the report's URL given the run's ID, name, and
+alias. The report links will appear in the External Links section of the Run or Project page (see figure {{figure}}).
+
+{% include userman-figure.md num=figure cap="Report links" img="site-config-external-links.png" %}
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Default Values" %}
+
+It is possible to set a default value for some fields to save users from repetitive entry. If a default value is
+configured, it will be filled in automatically when creating new items. The user is still free to change the value if
+they would like.
+
+The only field that can currently be configured with a default value is sample scientific name. This setting only
+affects the bulk entry form.
```

Comment:
It also affects the single entry form ( `/miso/sample/new/157` or any valid project number).

Reply:
Hmm, that is indeed very specific...